knative/describer.go

```diff
@@ -31,7 +31,7 @@ func NewDescriber(namespaceOverride string) (describer *Describer, err error) {
 // restricts to label-syntax, which is thus escaped. Therefore as a knative (kube) implementation
 // detal proper full names have to be escaped on the way in and unescaped on the way out. ex:
 // www.example-site.com -> www-example--site-com
-func (d *Describer) Describe(ctx context.Context, name string) (description fn.Description, err error) {
+func (d *Describer) Describe(ctx context.Context, name string) (description fn.Info, err error) {
```

Comment:
```
func (d *Describer) Describe(ctx context.Context, name string) (description fn.Info, err error) {
```
Shouldn't this be changed as well?

Reply:
I thought about that and decided not to change it for a couple of reasons. I meant to put this in the PR description, but just forgot.

* `Infoer` is a terrible name! :rofl: 
* The `Describer` in the `knative` package is responsible for the verb "describe" which the client uses to surface the noun "info" and they don't need to necessarily evolve 1:1 :sunglasses: 

But now by the light of morning, those reasons seem pretty weak. Do you have any suggestions for naming? I don't particularly like this.

```go
type Infoer struct {
  // ...
}

func (i *Infoer) Info(ctx context.Context, name string) (description fn.Info, err error)
```

What do you think of this?

```go
type Info struct {
  // ...
}

func (i *Info) Fetch(ctx context.Context, name string) (description fn.Info, err error)
```

Reply:
Hm, seems like a tradeoff either way.  I agree that `Describer` seems a lot nicer than `Infoer`, and that it's fine if the CLI and client library aren't exactly mapped, if it means the client library is closer to what we think is "correct".  I could go either way on it, but leaving it Describe fits for me:  run the `info` command to get a description.

Reply:
Hm, seems like a tradeoff either way.  I agree that `Describer` seems a lot nicer than `Infoer`, and that it's fine if the CLI and client library aren't exactly mapped, if it means the client library is closer to what we think is "correct".  I could go either way on it, but leaving it Describe fits for me:  run the `info` command to get a description.

Reply:
> Not to mention, there is the possibility that `info` may display more than just the description from the describer in the future...

Thanks for mentioning this. I avoided saying it because it's easy to fall down the flexibility hole.